templates/keycloak-update.sh

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
Pls. add this script to `./local_debug.sh`
https://github.com/eclipse-che/che-operator/blob/main/local-debug.sh#L54

Reply:
And in Dockerfile as well
https://github.com/eclipse-che/che-operator/blob/main/Dockerfile#L51